src/controller/index.ts

```diff
@@ -63,6 +65,8 @@ export default class SimulariumController {
         this.visData = new VisData();
         this.tickIntervalLength = 0; // Will be overwritten when a trajectory is loaded
         this.postConnect = () => noop;
+        this.startRecording = () => noop;
+        this.stopRecording = () => noop;
 
```

Comment:
sim controller methods are how we usually trigger viewer related actions in the front end